src/yang/ietf-qos-action.yang

```diff
@@ -629,6 +650,41 @@ module ietf-qos-action {
                 "Weighted Random Early Detect (WRED) configuration.";
             }
           }
+
+          case codel {
+            container codel {
+              uses codel-config-parameters;
+              leaf ecn-enabled {
+                type boolean;
+                default "false";
+                description
+                  "ECN is enabled on the queue.";
```

Comment:
Please update the description statement saying:

"When configure as true, ECN is enabled on the queue."